packages/back-end/src/models/ExperimentModel.ts

```diff
@@ -802,4 +805,37 @@ export const logExperimentDeleted = async (
   return emittedEvent.id;
 };
 
-// endregion Events
+export const getAllVisualExperiments = async (
+  organization: string
+): Promise<
+  Array<{
+    experiment: ExperimentInterface;
+    visualChangeset: VisualChangesetInterface;
+  }>
+> => {
+  const visualChangesets = await findVisualChangesets(organization);
+  const allVisualChanges = flatten(
+    visualChangesets.map((vc) => vc.visualChanges)
+  );
+  const experiments = (
+    await ExperimentModel.find({
```

Comment:
Instead of calling `ExperimentModel.find` directly, use the `findExperiments` method.  That method returns ExperimentInterface objects instead of ExperimentDocument objects, so it gets rid of all the Mongoose-specific fields.

Reply:
Done